app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,68 @@
+class ActivitiesController < ApplicationController
+
+ 
+  def index
+    #@activities = Activity.all
+    if params[:start_date] && params[:end_date]
+       @actvities = Activity.between(Date.strptime(params[:start_date]), Date.strptime(params[:end_date]))
```

Comment:
In my email I just sent, this is the place you would then update the strptime with the format string. Should be the same as the one in my email. If that throws errors for you, let me know, and take a screenshot of the form!

Replace Activity on this line with ```current_user.activities.between(...)```

The "..." is there for brevity. It's not the actual argument.